src/package/packageVersion.ts

```diff
@@ -35,98 +34,127 @@ import { getCreatePackageVersionCreateRequestReport } from './packageVersionCrea
 import { listPackageVersions } from './packageVersionList';
 import { list } from './packageVersionCreateRequest';
 
+type Package2Version = PackagingSObjects.Package2Version;
+
 Messages.importMessagesDirectory(__dirname);
 const messages = Messages.loadMessages('@salesforce/packaging', 'package_version');
+
+export const Package2VersionFields = [
```

Comment:
@shetzel I was planning on putting the fields array for Package2 in the interfaces file.